sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveExternalCatalog.scala

```diff
@@ -139,11 +139,6 @@ private[spark] class HiveExternalCatalog(conf: SparkConf, hadoopConf: Configurat
         s"as table property keys may not start with '$SPARK_SQL_PREFIX': " +
         invalidKeys.mkString("[", ", ", "]"))
     }
-    // External users are not allowed to set/switch the table type. In Hive metastore, the table
```

Comment:
can we add a comment here about why this was removed. Give some context about this was done during Hive deprecation to add parity.